examples/KeyWallet/Pkh2Pkh.purs

```diff
@@ -31,3 +35,5 @@ main = runKeyWalletContract_ \pkh lovelace unlock -> do
   txId <- submit bsTx
   logInfo' $ "Tx ID: " <> show txId
   liftEffect unlock
+  awaitTxConfirmedWithTimeout (wrap 120.0) txId
```

Comment:
Our timeout for tests is 50s. (see Test.Utils.interpret). Come to think of it, we can just use `awaitTxConfirmed` (sorry for misleading task description).